script/playbackstrategy/msestrategy.js

```diff
@@ -160,25 +160,51 @@ define('bigscreenplayer/playbackstrategy/msestrategy',
         DebugTool.info('Manifest validity changed. Duration is: ' + event.newDuration);
       }
 
+      function onStreamInitialised () {
+        emitPlayerInfo();
```

Comment:
Could we just have this function as the handler directly on the event? Happy if you prefer this as more readable though.

Reply:
Nope, it has another usage.